src/main/cpp/src/utils/vid_mapper_pb.cc

```diff
@@ -368,6 +368,9 @@ int VidMapper::parse_infofields_from_vidmap(
         ref.m_length_descriptor.set_vcf_delimiter(i,
             vid_map_protobuf->fields(pb_field_idx).vcf_delimiter(i).c_str());
     }
+    ref.m_disable_remap_missing_with_non_ref = 
+      vid_map_protobuf->fields(pb_field_idx).has_disable_remap_missing_with_non_ref() ?
+      vid_map_protobuf->fields(pb_field_idx).disable_remap_missing_with_non_ref() : false;
```

Comment:
Wondering why tests/golden_outputs/t0_1_2_all_asa_java_query_vcf_no_remap_missing does not exercise this code or is it a codecov quirk? Otherwise is it easy enough to write a test  to cover this logic?

Reply:
The CI test uses the query json while this is protobuf. I'll add a test for this